llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2511,8 +2560,7 @@ class VPWidenCanonicalIVRecipe : public VPSingleDefRecipe {
 };
 
 /// A recipe for converting the canonical IV value to the corresponding value of
-/// an IV with different start and step values, using Start + CanonicalIV *
-/// Step.
+/// an IV with different start and step values, using Start + IV * Step.
```

Comment:
The line above also needs updating. Technically the operands now don't have to be IVs (which typically should have a constant step), it just adjust an input now.